### ISSUE TITLE
Needs to be tweaked

### ISSUE THREAD

**Issue by Cybesjie** [alfredapp/updated-third-party-python2-workflows#21]
### Workflow name

I Sheet You Not

### Workflow page

https://www.alfredforum.com/topic/9469-i-sheet-you-not-plug-excel-into-alfred/#comment-47102

**Comment by Cybesjie** [comment]
The latest version was released here: https://github.com/deanishe/i-sheet-you-not/releases

**Comment by vitorgalvao** [comment]
Why does it need to be tweaked? If you don’t explain what issue you’re bumping into, it’s impossible to help you.

**Comment by Cybesjie** [comment]
Got you. Here is the description: 

I followed the the knowledge base article to reinstall Python. I believe Python 2 is properly installed in my Mac. Still, the workflow doesn’t work.

I triggered the workflow in debug mode for a couple of times and I received the same error code:

ERROR: I Sheet You Not[Script Filter] Code 126: /Users/xin/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/1AA6D7C1-562E-4DAE-BD98-D8F930CE3986: ./isyn: /usr/bin/python: bad interpreter: No such file or directory


MacOS 12.3 (21E230)
Alfred4.6.1 [1274]
